solidity/dashboard/src/components/KeepOnlyPool.jsx

```diff
@@ -139,11 +141,37 @@ const KeepOnlyPool = ({
               &nbsp;KEEP
             </h4>
           </div>
-
+          <Banner
+            className="liquidity-info-banner liquidity-info-banner--warning mt-2"
+            style={{ maxWidth: "100%" }}
+          >
+            <Banner.Icon
+              icon={Icons.Warning}
+              className="liquidity-info-banner__icon liquidity-info-banner__icon--warning"
+            />
+            <div className="liquidity-info-banner__content">
+              <Banner.Title className="text-grey-60">
+                Incentives removed
+              </Banner.Title>
+              <Banner.Description className="liquidity-info-banner__content__description text-grey-60">
+                The incentives for this pool has been removed and you can no
+                longer deposit the KEEP tokens. You can still withdraw deposited
+                KEEP tokens and rewards that you already earned.&nbsp;
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={LINK.proposals.shiftingIncentivesToCoveragePools}
+                  className="text-grey-60"
```

Comment:
Can we also add `text-link` class here so the style of the "More info" link is similar to the ones in TBTC + Saddle and KEEP + TBTC banners?

Reply:
06f0146